mtgjson5/resources/cardkingdom_sealed_name_mapping.json

```diff
@@ -0,0 +1,204 @@
+{
+    "editions": {
+        "2017 gift pack": "gift pack 2017",
```

Comment:
could we make this json as a single children type? right now you can have string:string or string:array_of_strings which is confusing and hard to unmarshal - making it always string:array_of_string makes it simpler IMO

Reply:
I ended up removing the arrays because it was a hokey fix for something that should be resolved on a bigger level. See https://github.com/mtgjson/mtgjson/issues/1070.